tfx/conftest.py

```diff
@@ -0,0 +1,8 @@
+"""Test configuration."""
+from absl import flags
+
+def pytest_configure(config):
+  del config
```

Comment:
This deletes the pytest `Config` object - how come this is needed here? Won't this destroy all pytest and pytest plugin configuration?

Reply:
I initially thought the config object was only used within the function, but to prevent potential issues, I've removed it. Thanks!